### PR TITLE
added kitchen meta.yaml file

### PR DESCRIPTION
**Pull request by saibhargav1508**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by saibhargav1508** [comment]
What is the flow of a Pull Request after the builds pass? Should the PR be closed before reviewing or will the reviewer close it after merging?
Please don't mind the question, just curious to know.

File: recipes/kitchen/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - kitchen
+    - kitchen.collections
```
Comment (synapticarbors) [inline recipes/kitchen/meta.yaml:28]: Check all submodules in imports test

```yaml
- kitchen
- kitchen.versioning
- kitchen.i18n
- kitchen.iterutils
- kitchen.collections
- kitchen.text
- kitchen.pycompat24
- kitchen.pycompat24.base64
- kitchen.pycompat24.sets
- kitchen.pycompat25
- kitchen.pycompat25.collections
- kitchen.pycompat27
- kitchen.pycompat27.subprocess
```

**Comment by synapticarbors** [comment]
Once the PR has been reviewed and passes all of the CI, then one of the reviewers will merge the PR, at which point the feedstock repo will be built:

https://conda-forge.org/docs/maintainer/00_intro.html

**Comment by saibhargav1508** [comment]
The windows build seems to be failing because of this:
AttributeError: module 'os' has no attribute 'sysconf'
@synapticarbors Should I restrict the build only to Linux and MAC?

**Comment by synapticarbors** [comment]
So according the the metadata in the `setup.py`, this package should be os independent. That said, this looks like a bug wherein the call is in a `try/except`, but it raises an `AttributeError`, but only checks for a `ValueError`. I think we can just ignore it

File: recipes/kitchen/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    - kitchen.pycompat27
+    - kitchen.pycompat27.subprocess
+
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://fedorahosted.org/kitchen
```
Comment (synapticarbors) [inline recipes/kitchen/meta.yaml:49]: This link doesn't work any more. It looks like the code lives at https://github.com/fedora-infra/kitchen currently, so maybe change to that.

Comment (saibhargav1508) [reply recipes/kitchen/meta.yaml]: Link updated

**Comment by saibhargav1508** [comment]
Thanks for the help and suggestions @synapticarbors

**Pull request merged**